src/ocamlorg_frontend/pages/home.eml

```diff
@@ -1,10 +1,12 @@
 let render () =
 Layout.render
+~(release : Data.Release.t)
 ~use_swiper:true
 ~title:"Welcome to a World of OCaml"
 ~description:"OCaml is a general-purpose, industrial-strength programming language with an emphasis on expressiveness and safety."
 ~canonical:""
 ~banner:true @@
+~(release : Data.Release.t)
```

Comment:
I think it `release` needs to be a parameter of the `render` function in line 1, instead.

Reply:
I use iOS. i am afraid i can’t compile the code. Anyway if there is anything i can do please let me know.

Reply:
Sorry for the inconvenience. I will try to compile code. If I need anything I will get back to you.